flake.nix

```diff
@@ -180,7 +182,7 @@
         server-vm = inputs.nixos-generators.nixosGenerate {
           inherit pkgs;
           format = "vm";
-          modules = [ config.global config.qemu ];
+          modules = [ config.global config.qemu config.nixbuild ];
```

Comment:
If you forget to copy over `nixbuild.pem`, I guess no errors occur, but the vm just doesn't use nixbuild.net to build?

Reply:
Yeah, it'll just say it can't access `eu.nixbuild.net` and build locally